public/javascripts/SVValidate/src/keyboard/Keyboard.js

```diff
@@ -82,10 +82,55 @@ function Keyboard(menuUI) {
         }
     }
 
+    function _setSeverity(severity) {
+        console.log(severity);
+        if(svv.newValidateBeta){
+            svv.ui.newValidateBeta.severityMenu.find(`#severity-button-${severity}`).click();
+        } else {
+            menuUI.severityMenu.find('.severity-level').removeClass('selected');
```

Comment:
You can't set severity in the regular validate page, so we shouldn't be doing anything in this case. There's no `severityMenu` as part of `menuUI` in the regular Validate case. Not sure where this is coming from.

Reply:
This is still an issue